install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -28,6 +28,28 @@ security:
   usernameKey: username
   passphraseKey: passphrase
 nodeSelector: {}
+
+# Specify the pod anti-affinity that allows you to constrain which nodes
+# your pod is eligible to be scheduled based on labels on pods that are
+# already running on the node rather than based on labels on nodes.
+# There are currently two types of anti-affinity:
+#    "requiredDuringSchedulingIgnoredDuringExecution"
+#    "preferredDuringSchedulingIgnoredDuringExecution"
+# which denote “hard” vs. “soft” requirements, you can define your values
+# in "podAntiAffinityLabelSelector" and "podAntiAffinityTermLabelSelector"
+# correspondingly.
+# For example:
+# podAntiAffinityLabelSelector:
+# - key: security
+#   operator: In
+#   value: S1
```

Comment:
should this be values?

Reply:
also does the key or operator matters to our users?  any reason why user needs to configure this?